cspell.yaml

```diff
@@ -0,0 +1,70 @@
+version: "0.2"
+language: en
+allowCompoundWords: true
+dictionaries:
+  - node
+  - typescript
+words:
+  - autorest
+  - azsdkengsys
+  - azurecr
+  - blockful
+  - blockless
+  - cadl
+  - Contoso
+  - CRUDL
+  - dogfood
+  - eastus
+  - esbuild
+  - globby
+  - inmemory
+  - instanceid
+  - intrinsics
+  - jsyaml
+  - msbuild
+  - MSRC
+  - munge
+  - mylib
+  - nostdlib
+  - oapi
+  - oneof
+  - onig
+  - onigasm
+  - onwarn
+  - openapi
+  - openapiv
+  - picocolors
+  - proto
+  - protobuf
+  - protoc
+  - regen
+  - rpaas
+  - rushx
+  - safeint
+  - strs
+  - TSES
+  - Uhoh
+  - unassignable
+  - Uncapitalize
+  - uninstantiated
+  - unioned
+  - unprojected
+  - unsourced
+  - unversioned
+  - vsix
+  - vswhere
+  - Vtbzpw
+  - westus
+  - xplat
+  - 𐌰𐌲
+  - 𐌰𐌲𐌰𐌲𐌰𐌲
```

Comment:
Maybe cspell disable next line for these test cases

Reply:
was a lot of those. But I guess I can do a file exclusion like mike did in his pr
<img width="274" alt="image" src="https://user-images.githubusercontent.com/1031227/189179871-20c64f4a-8bbf-438c-8597-e6fcb14e3362.png">

Reply:
updated to use file ignore